source/hexo.md

```diff
@@ -26,6 +26,9 @@ title: 在Hexo 中使用
 ## cactus-dark
 > [Open](https://github.com/probberechts/cactus-dark/pull/54) `latest`
 
+## cactus-white-cn
```

Comment:
```suggestion
## cactus-white-cn
```

Please add it to the [English document](https://github.com/xCss/Valine-docs/blob/master/source/en/hexo.md)